lib/rules/no-comment-textnodes.js

```diff
@@ -19,7 +19,7 @@ module.exports = function(context) {
 
   return {
     Literal: function(node) {
-      if (/\s*\/(\/|\*)/.test(node.value)) {
+      if (/^\s*\/(\/|\*)/m.test(node.value)) {
```

Comment:
ah right, i'm assuming the text node will be split up by lines, and that's not likely to be the case.

Could we add your example as a test, to prevent a regression?